data/49-io.elementary.initial-setup.pkla

```diff
@@ -4,3 +4,10 @@ Action=org.freedesktop.accounts.user-administration;io.elementary.pantheon.Accou
 ResultAny=yes
 ResultInactive=yes
 ResultActive=yes
+
+[Host name]
+Identity=unix-user:lightdm
+Action=org.freedesktop.hostname1.set-hostname;org.freedesktop.hostname1.set-static-hostname;
```

Comment:
```suggestion
Action=org.freedesktop.hostname1.set-pretty-hostname;org.freedesktop.hostname1.set-static-hostname;
```

I'm not familiar with the pkla file format at all, unfortunately, but that might work.

Reply:
The current value is fine, see here for possible values: https://www.freedesktop.org/software/systemd/man/org.freedesktop.hostname1.html#Security